src/app/visit/visit.cpp

```diff
@@ -13,10 +14,13 @@ Visit::~Visit()
     removeFromExtent(this);
 }
 
-std::shared_ptr<Visit> Visit::createVisit(std::shared_ptr<Doctor> doctor)
+std::shared_ptr<Visit> Visit::createVisit(std::shared_ptr<Doctor> doctor, std::shared_ptr<Patient> patient)
 {
     auto visit = std::shared_ptr<Visit>(new Visit());
+    visit->doctor_association_ = doctor;
+    visit->patient_ = patient;
     doctor->addVisitAssociation(visit);
+    patient->addVisit(visit);
     return visit;
 }
```

Comment:
In uml, if it's not composition, we should create function addPatient to add its association. Creating visit with doctor using static function means that we deal with composition and that's the only way we should use this function, to initialise it with doctor straight away. (You can make issue out of it, and I'll fix it)

Reply:
reverted